src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -204,6 +209,20 @@ const SearchResult = ({
           </div>
         </dd>
       ) : null}
+      {showPDFInfoVersionThree ? (
```

Comment:
```suggestion
      {showPDFInfoVersionThree && (
```

Reply:
If you want I think you can just do this with && instead of a ternary 🙂

Reply:
Oh I forgot that is a think you can do since the Right will always be true.
Then it just depends on the variable on the left.